README.md

```diff
@@ -1,9 +1,13 @@
 ## [WDL](https://software.broadinstitute.org/wdl/) syntax for Sublime Text editor
 
-This repo provides support for the [Workflow Development Language (WDL)](https://software.broadinstitute.org/wdl/) in the [Sublime Text Editor](https://www.sublimetext.com/).
+This repo provides support for the [Workflow Development Language (WDL)](https://software.broadinstitute.org/wdl/) in the [Sublime Text Editor](https://www.sublimetext.com/) 
+and [Visual Studio Code](https://code.visualstudio.com/).
 
-Installation:
+ ### Installation:
+ #### Sublime Text:
   * Install [Package Control](https://packagecontrol.io/) package manager for Sublime via [these instructions](https://packagecontrol.io/installation).
   * Use <kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> on OSX or <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> on Linux to open Command Palette, then select `Package Control: Install Package`.
   * Search for "WDL Syntax" and hit <kbd>Enter</kbd>. 
-  
+ 
+ #### Visual Studio Code:
+ * Go to Extensions and search for WDL Syntax Highlighter and install!
```

Comment:
In a world where we don't have a MS account, is there any way to manually install a pre-packaged release?

Reply:
I will look into finding a simple way and update this doc.